ccd-definition/AuthorisationCaseEvent/solicitor.json

```diff
@@ -33,5 +33,12 @@
     "CaseEventID": "ACKNOWLEDGE_SERVICE",
     "UserRole": "caseworker-civil-solicitor",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "DEFENDANT_RESPONSE",
```

Comment:
should this not be RESPOND_TO_CLAIM ?

Reply:
As claimant will also be responding after this. So was thinking to differentiate those two i.e. `DEFENDANT_RESPONSE` and `CLAIMANT_RESPONSE `.

Reply:
Are you sure both events are "respond to the claim"? i.e is one responding to the claim and the other responding to that response?

Eitherway I still think response is wrong. Other events would mean this should follow RESPOND_TO_CLAIM, maybe with DEFENDANT / RESPONDENT at the front

Reply:
I will leave the naming decision to @AlisonRevittHM